signatures/FALCON/FALCON/bench/avx2_512_Skylake.yaml

```diff
@@ -0,0 +1,7 @@
+platform: Intel Core i7-6567 (Skylake) 3.3 GHZ
+timings:
+  unit: cycles 
+  gen:  26604000
+  enc|sign: 531684
```

Comment:
The timings used are sometimes from the `(tree, ct)` line, sometimes from `(dynamic, ct)`. I didn't really find out what the differences is from skimming the specs, did you?

Here, it's tree

Reply:
I also did not find a detailed explanation of "dynamic". I ,however, changed all the results to (tree, ct) to be consistent.